src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -610,10 +656,19 @@ internal unsafe struct ModuleFixupCell
         [StructLayout(LayoutKind.Sequential)]
         internal unsafe struct MethodFixupCell
         {
+            private const int CharSetMask = 0x7;
+            private const int IsObjectiveCMessageSendMask = 0x8;
+            private const int ObjectiveCMessageSendFunctionMask = 0x70;
+            private const int ObjectiveCMessageSendFunctionShift = 4;
```

Comment:
Is there no way to share these with the constants in `PInvokeMethodFixupNode.cs`?

Reply:
They can be placed into src/coreclr/tools/Common/Internal/Runtime/RuntimeConstants.cs that is included from both the compiler and CoreLib and serves this purpose.

Reply:
I moved these to `RuntimeConstants.cs`.